### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Zibilyonik**

### CODE REVIEW

File: spec/enumerable_spec.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
+require './enumerable'
```
Comment (mosaaleb) [inline spec/enumerable_spec.rb:1]: When I run the `rspec` it gives me an arror `cannot load such file -- ./enumerable`
- Hint: file name is `Enumerable` not `enumerable`

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Well done @Zibilyonik :clap:

#### All test cases are good and passed. However an [error](https://github.com/Zibilyonik/Enumerable-rspec/pull/1#discussion_r312768456) appeared when I tried to run rspec.



- [ ] There is a lot rubocop problems in your code. Have a look at [style guidelines](https://github.com/rubocop-hq/ruby-style-guide) and fix these errors. We also need to enable [`stickler`](https://github.com/microverseinc/linters-config/blob/master/ruby/) for this repo.

**TIP** You can install rubocop plugin in your preferred text editor as it will catch the errors once they appear while still in development stage.

Screenshot from 2019-08-12 04-28-50


**Friendly reminder**: You should use Git Flow in this project -- your branch should have more descriptive name like `add-tests`. However, changing it now would take too much time. Please make sure you use Github/Git Flow in the next project. You will find instructions in the Patwright lesson description.

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
Hello @Zibilyonik ,

You're just linter errors away from getting a thumb up. Your tests are good and pass well. 

image
Please click on the details to see all the linting errors you should fix. It's important to enforce linter corrections as it makes your code look professional :sunglasses:

Please fix them and request for one more re-review.

Happy coding:v:

**P.S.**

This is a collaborative project. Any reason why you're doing it alone?

**Comment by Zibilyonik** [comment]
> Hello @Zibilyonik ,
> 
> You're just linter errors away from getting a thumb up. Your tests are good and pass well.
> 
> image
> Please click on the details to see all the linting errors you should fix. It's important to enforce linter corrections as it makes your code look professional 😎
> 
> Please fix them and request for one more re-review.
> 
> Happy coding✌️
> 
> **P.S.**
> 
> This is a collaborative project. Any reason why you're doing it alone?

Most of those lint errors are single-double quote issues which are only based on user choice and not style guidelines. I will deal with the rest of the issues now.

**Review by maelfosso — CHANGES_REQUESTED** [review]
Hello @Zibilyonik 

Please, Ruby has coding rules. Stickler-ci is here to help students to be good in Ruby, to master the coding rules. Please, take times to correct the stickler errors.

Like @JamesOkunlade ask you up, this project is a Collaborative project, why are working alone?

Thanks

**Comment by Zibilyonik** [comment]
> Hello @Zibilyonik
> 
> Please, Ruby has coding rules. Stickler-ci is here to help students to be good in Ruby, to master the coding rules. Please, take times to correct the stickler errors.
> 
> Like @JamesOkunlade ask you up, this project is a Collaborative project, why are working alone?
> 
> Thanks

If my partner hadn't left the course and left me alone to deal with 4 projects at once, I would be dealing with those minor issues AND working with my partner at the same time. But since that is not the case, I have to finish these projects as fast as possible and do them alone at the same time.

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Hello @Zibilyonik! 
Please consider correcting your linter errors before requesting a review. The linter is meant to guide your development process and when you get into the real world, this will be no different as companies adhere strictly to their style guides.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi @Zibilyonik 👋

Good job on this you've done quite well.👌

- [x] - Your spec test for my_map throws an error: `no implicit conversion of integer into Hash`. Kindly fix this.

- [x] - In addition the it block for my_map test says: `'should loop through the array and determine if any element is true'`, however please fix this because that is not what you are testing for in the expect block.

- [ ] OPTIONAL - Sorry this wasn't mentioned by previous reviewers but you should only have one expect per it block. You can consider correcting this.

- [x] Also it would be nice if you could add instructions to run the tests on your readMe.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Review by lucasmazo32 — APPROVED** [review]
# Project approved :rocket:

#### Great job!

#### Your project looks great!

## Before you merge this PR

Friendly reminder: Please make the PR from a feature branch to the development branch using git-flow for your next projects. 

### You can go ahead and merge this pull request!

### Happy coding! :computer:

## Lucas Mazo

**Pull request merged**